static/app/views/integrationOrganizationLink/index.tsx

```diff
@@ -273,9 +295,42 @@ export default class IntegrationOrganizationLink extends DeprecatedAsyncView<
       ),
     }));
 
+    let integrationAlert;
+    if (this.integrationSlug === 'github') {
```

Comment:
Good point. Actually, if we don't get any data, this could mean that attacker waited out the period of data exposure (10 minutes) and sent the link, so we really need to warn about this. Something like, "We could not verify the authenticity of the installation request. You may want to restart the installation process." Sounds good?

Reply:
That's a good call, it may be worth changing the alert type from `info` to `warning` or `error` as well to call specific attention to it.